common.py

```diff
@@ -97,14 +60,15 @@ def draw_landmarks_on_binary_image(rgb_image, detection_result):
 
 def plot_face_blendshapes_bar_graph(face_blendshapes):
     # Extract the face blendshapes category names and scores.
-    face_blendshapes_names = [
-        face_blendshapes_category.category_name for face_blendshapes_category in face_blendshapes]
-    face_blendshapes_scores = [
-        face_blendshapes_category.score for face_blendshapes_category in face_blendshapes]
+    face_blendshapes_names, face_blendshapes_scores = [], []
+    for face_blendshapes_category in face_blendshapes:
+        face_blendshapes_names.append(face_blendshapes_category.category_name)
+        face_blendshapes_scores.append(face_blendshapes_category.category_score)
+
```

Comment:
Not a big difference in performance, but if you can avoid to access two time the same list, dict, etc... is good to do it :)